src/sysupdate/sysupdate-resource.c

```diff
@@ -278,17 +279,11 @@ static int download_manifest(
         assert(ret_buffer);
         assert(ret_size);
 
-        /* Download a SHA256SUMS file as manifest */
-
-        r = import_url_append_component(url, "SHA256SUMS", &suffixed_url);
```

Comment:
i think it would be nicer to keep this in place here, but simply take one more parameter to download_manifest(), and then do `manifest ?: "SHA256SUMS"`